include/zenoh-pico/config.h

```diff
@@ -16,9 +16,9 @@
 #define INCLUDE_ZENOH_PICO_CONFIG_H
 
 /*--- CMake generated config; pass values to CMake to change the following tokens ---*/
-#define Z_FRAG_MAX_SIZE 4096
-#define Z_BATCH_UNICAST_SIZE 2048
-#define Z_BATCH_MULTICAST_SIZE 2048
+#define Z_FRAG_MAX_SIZE 300000
+#define Z_BATCH_UNICAST_SIZE 65535
+#define Z_BATCH_MULTICAST_SIZE 8096
```

Comment:
These values should not be modified.

Reply:
Thanks, seems it was committed by mistake